src/vue/vuex-modules/data/auth-module.js

```diff
@@ -55,7 +56,12 @@ AuthModule.store = (moduleInstance) => {
         state.userNickName = ''
       },
       setNotification (state, data) {
-        state.notification.visible = true
+        // don't show login notifications if they have been hidden
```

Comment:
Maybe it's better here and in the following method to describe the data object in JSDoc? At least here data is a complex object so it's good to know what props are required and what their meanings are.

Reply:
that's a good idea. Will do that.